cpu/cortexm_common/Makefile.include

```diff
@@ -70,8 +70,12 @@ else
 endif
 
 # Configure riotboot bootloader and slot lengths
-# 4KB are currently enough
-RIOTBOOT_LEN ?= 0x1000
+# 4KB are currently enough, set it to 16KB if USB-DFU is used
+ifneq (,$(filter usbus_dfu,$(USEMODULE)))
+  RIOTBOOT_LEN ?= 0x4000
```

Comment:
While this is sufficient to also fit the bootloader on the nrf52840dk, the nrf52 MCU's have their own override for this so that it is at least 2 pages. This means that this override won't be applied to the nrf52 MCU's.

Reply:
Can we use "??=" as weak assignment in Makefile world ?

Reply:
Anyways, I pushed a fix for that too.